modules/samples/default/decision-tree/README.MD

```diff
@@ -7,7 +7,9 @@ Algorithm: Decision-Tree
 Before Running
 ==============
 
-* Start WSO2 Machine Learner product with a fresh database.
+* Start WSO2 Machine Learner product.
+
+* Make sure jq is installed.
```

Comment:
@manoramahp thanks. We don't have any other alternative right? If it is the case, can we at least give a link to how to install it etc. for different OSs.